mea/Cargo.toml

```
@@ -13,32 +13,47 @@
 # limitations under the License.
 
 [package]
+authors = ["tison <wander4096@gmail.com>"]
+categories = ["async", "synchronization", "waitgroup", "concurrency", "no-std"]
+description = """
+A runtime-agnostic library providing essential synchronization primitives for asynchronous Rust programming.
+Includes Barrier, Latch, Mutex, Semaphore, and WaitGroup implementations.
+"""
+documentation = "https://docs.rs/mea"
+keywords = ["async", "concurrency", "synchronization", "waitgroup", "mutex"]
 name = "mea"
+readme = "README.md"
```

```suggestion
readme.workspace = true
```

Should use this one to correctly resolve the path?